src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -172,7 +172,7 @@ export const ArtworkAuctionCreateAlertHeaderFragmentContainer = createFragmentCo
         isInAuction
         artistNames
         internalID
-        artists {
+        artists(shallow: true) {
```

Comment:
...especially now that this is about to spread, and future devs won't be able to understand what this means, given the fact that most current devs don't (actually) understand what this abstraction means

Reply:
#nonblocking